README.md

```diff
@@ -237,7 +237,8 @@ begin
   webauthn_credential.verify(
     session[:authentication_challenge],
     public_key: stored_credential.public_key,
-    sign_count: stored_credential.sign_count
+    sign_count: stored_credential.sign_count,
+    user_verification: true, # needed for passwordless verification
```

Comment:
I think we should also add this to creation options in the [initiation phase of the Credential Registration ceremony](https://github.com/cedarcode/webauthn-ruby#initiation-phase) 🙂